table_top:
  length: 0.72
  width: 0.72
  height: 0.15

leg1:
  length: 0.05
  width: 0.05
  height: 0.3

leg2:
  length: 0.05
  width: 0.05
  height: 0.3

leg3:
  length: 0.05
  width: 0.05
  height: 0.3

leg4:
  length: 0.05
  width: 0.05
  height: 0.3